src/test/scala-2.11/markets/participants/MarketParticipantSpec.scala

```diff
@@ -52,39 +52,25 @@ class MarketParticipantSpec extends TestKit(ActorSystem("MarketParticipantSpec")
 
   feature("A MarketParticipant actor should be able to add and remove outstanding orders.") {
 
-    val tradable = Security("GOOG")
-
-    scenario("A MarketParticipant actor receives a Canceled message...") {
-
-      val marketParticipant = TestActorRef(new TestMarketParticipant)
-
-      Given("A MarketParticipant actor with outstanding orders...")
-      val order = LimitAskOrder(marketParticipant, 10, 100, 1, tradable, uuid())
-      val marketParticipantActor = marketParticipant.underlyingActor
-      marketParticipant.underlyingActor.outstandingOrders += order
-
-      When("A MarketParticipant actor receives a Canceled message...")
-      val canceled = Canceled(order, 3, uuid())
-      marketParticipant ! canceled
-
-      Then("...it should remove the canceled order from its outstanding orders.")
-      marketParticipantActor.outstandingOrders.headOption should be(None)
-
-    }
+    val tradable = TestTradable("GOOG")
+    val market = TestProbe()
+    val markets = mutable.Map[Tradable, ActorRef](tradable -> market.ref)
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [It's preferred to use immutable collections instead of mutable ones](https://www.codacy.com/app/davidrpugh/markets-sandbox/file/1704906920/issues/source?bid=2921361&fileBranchId=3046267#l57)